ee/codegen_integration/fixtures/simple_inline_subworkflow_node/code/display/nodes/subworkflow_node/__init__.py

```diff
@@ -1,10 +1,12 @@
 from uuid import UUID
 
 from vellum_ee.workflows.display.nodes import BaseInlineSubworkflowNodeDisplay
-from vellum_ee.workflows.display.nodes.types import PortDisplayOverrides
+from vellum_ee.workflows.display.nodes.types import NodeOutputDisplay, PortDisplayOverrides
 from vellum_ee.workflows.display.vellum import NodeDisplayData, NodeDisplayPosition
 
 from ....nodes.subworkflow_node import SubworkflowNode
+from .nodes import *
```

Comment:
@noanflaherty @NgoHarrison Codegen: We need to import the displays for the subworkflow to register the overrides

Reply:
Yep, this indeed makes sense! In fact, we probably also need to include a `from .workflow import *` so that the inner workflow's display class is in the import path.

Please feel free to remove this case from codegen tests and cut a card in my epic to add back after I've added codegen support for this piece and any others this PR exposes.

Reply:
I have `from .workflow import *` just below this line - doesn't fix the subworkflow node output ID issue :/